[INFO   ] Logger: Record log in /storage/emulated/0/learn kivy/16-progressbar/.kivy/logs/kivy_22-12-18_39.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  6 2021, 01:34:26) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 V@0530.0 (GIT@8382da08aa, I139f36ad46, 1645197494) (Date:02/18/22)'>
[INFO   ] GL: OpenGL vendor <b'Qualcomm'>
[INFO   ] GL: OpenGL renderer <b'Adreno (TM) 619'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/16-progressbar/main.kv", line 41, in <module>
[WARNING] stderr:     value:menu_slidr.value
[WARNING] stderr: NameError: name 'menu_slidr' is not defined
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 695, in _apply_rule
[WARNING] stderr:     value, bound = create_handler(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 245, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/learn kivy/16-progressbar/main.kv", line 41:
[WARNING] stderr: ...
[WARNING] stderr:      39:        ProgressBar:
[WARNING] stderr:      40:            max:100
[WARNING] stderr: >>   41:            value:menu_slidr.value
[WARNING] stderr: ...
[WARNING] stderr: NameError: name 'menu_slidr' is not defined
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/16-progressbar/main.kv", line 41, in <module>
[WARNING] stderr:     value:menu_slidr.value
[WARNING] stderr: 
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/16-progressbar/main.py", line 27, in <module>
[WARNING] stderr:     Main().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 918, in _run_prepare
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 691, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 306, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 406, in load_string
[WARNING] stderr:     widget.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 463, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 541, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 710, in _apply_rule
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/learn kivy/16-progressbar/main.kv", line 41:
[WARNING] stderr: ...
[WARNING] stderr:      39:        ProgressBar:
[WARNING] stderr:      40:            max:100
[WARNING] stderr: >>   41:            value:menu_slidr.value
[WARNING] stderr: ...
[WARNING] stderr: BuilderException: Parser: File "/storage/emulated/0/learn kivy/16-progressbar/main.kv", line 41:
[WARNING] stderr: ...
[WARNING] stderr:      39:        ProgressBar:
[WARNING] stderr:      40:            max:100
[WARNING] stderr: >>   41:            value:menu_slidr.value
[WARNING] stderr: ...
[WARNING] stderr: NameError: name 'menu_slidr' is not defined
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/16-progressbar/main.kv", line 41, in <module>
[WARNING] stderr:     value:menu_slidr.value
[WARNING] stderr: 
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 695, in _apply_rule
[WARNING] stderr:     value, bound = create_handler(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 245, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: 
